modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -85,17 +87,63 @@ object ElasticRequest {
         Right(None)
     }
 
-  def search[A](index: IndexName, query: ElasticQuery[_])(implicit
+  def search(
+    index: IndexName,
+    aggregation: ElasticAggregation
+  ): ElasticRequest[ScalaMap[String, AggregationResponse], Search] =
```

Comment:
Does it make sense to introduce `ElasticAggregations` or `Aggregations` that will hide the `Map` implementation?